src/Raven.Client/Documents/Commands/Batches/DeleteCommandData.cs

```diff
@@ -9,24 +9,38 @@ namespace Raven.Client.Documents.Commands.Batches
     public class DeleteCommandData : ICommandData
     {
         public DeleteCommandData(string id, string changeVector)
+            :this(id, changeVector, changeVector)
```

Comment:
I'm worried about users that are using that explicitly.
We encourage that when using `Defer`, so that is part of the interface that we care about.
`OldChangeVector` is mostly not something that I expect users to really notice or care about.

Reply:
OK - makes sense